integration-tests/tests/delete-orphaned-workspaces.js

```diff
@@ -61,20 +64,20 @@ const deleteOrphanedWorkspaces = withUserToken(async ({ page, testUrl, token })
     const failedNames = failedDeletes.map(({ name, cloudPlatform }) => `${name} (${cloudPlatform})`).join(', ');
     console.info(`Triggered delete on workspaces: ${deletedNames}`);
     if (failedNames) {
-      console.warn(`Failed to delete workspaces: ${failedNames}`);
+      console.warn(`Failed to delete ${failedDeletes.length} workspaces : ${failedNames}`);
     }
 
     const currentOrphans = await listOrphanWorkspaces(page, { isVerbose: false });
     const persistentOrphans = currentOrphans.filter(({ workspace: { name } }) =>
       oldWorkspaces.some(({ workspace: { name: oldName } }) => oldName === name)
     );
+    console.log(`${persistentOrphans.length} persistent orphans remaining after delete attempt.`);
     const deleteFailedOrphans = getDeleteFailedWorkspaceNames(persistentOrphans);
     const newlyFailedDeletes = deleteFailedOrphans.filter((newName) => !oldWorkspaceNamesInDeleteFailed.includes(newName));
 
     if (newlyFailedDeletes.length) {
-      const newlyFailedNames = newlyFailedDeletes.map(({ name, cloudPlatform }) => `${name} (${cloudPlatform})`).join(', ');
```

Comment:
This wouldn't have worked because `newFailedDeletes` doesn't have a `cloudPlatform`, it's just the name; now that I've included the namespace along with the name that should help